vars/family_Debian.yml

```diff
@@ -1 +1,2 @@
-aem_dispatcher_module_path: /usr/lib/apache2/modules
\ No newline at end of file
+aem_dispatcher_module_path: /usr/lib/apache2/modules
+_aem_dispatcher_apache_server_root: "{{ aem_dispatcher_apache_server_root | default('/etc/apache2') }}"
```

Comment:
Doesn't that need to default to `apache_server_root` before defaulting to a fixed value to keep the current behaviour?

Reply:
I don't think that this is necessary. If the dependency can be disabled the apache_server_root will also not be available. The value is set in the same way as in the wcm_io_devops.apache role.

Reply:
I added `apache_server_root` as default value.